### PR TITLE
Fix Jenkins build

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

**Comment by aelred** [comment]
Confirmed this works via elephant

File: Jenkinsfile [hunk 1]
```diff
@@ -251,7 +251,8 @@ def runBuild() {
                 stage('Run the benchmarks') {
                     mvn "clean test -P janus -Dtest=*Benchmark -DfailIfNoTests=false -Dcheckstyle.skip=true -Dfindbugs.skip=true -Dpmd.skip=true"
                     archiveArtifacts artifacts: 'grakn-test/test-integration/benchmarks/*.json'
-		    archiveArtifactsS3 artifacts: 'grakn-test/test-integration/benchmarks/*.json'
```
Comment (fppt) [inline Jenkinsfile]: TODO?

Comment (aelred) [reply Jenkinsfile]: 🔽

Comment (fppt) [reply Jenkinsfile]: Oh you want to fix that in another PR? Sorry did not get that. Thought maybe it was a missed todo.

**Pull request merged**